nidm/nidm-experiment/imports/iao_import.ttl

```diff
@@ -1,6 +1,6 @@
+@prefix iao: <http://purl.obolibrary.org/obo/iao/2015-02-23/> .
```

Comment:
Looking closer at IAO, it looks like the namespace URL is `http://purl.obolibrary.org/obo/iao.owl#` (cf. https://github.com/information-artifact-ontology/IAO/blob/master/releases/2017-03-24/iao.owl#L2). Or did you find this one in a more recent source?